localstack/utils/collections.py

```diff
@@ -414,6 +414,11 @@ def is_none_or_empty(obj: Union[Optional[str], Optional[list]]) -> bool:
     )
 
 
-def select_from_typed_dict(typed_dict: Type[TypedDict], obj: Dict):
+def select_from_typed_dict(typed_dict: Type[TypedDict], obj: Dict, filter: bool = False) -> Dict:
     """Select a subset of attributes from a dictionary based on the keys of a given `TypedDict`"""
```

Comment:
nit: maybe we could add proper pydoc here to also disambiguate the `filter` term